### ISSUE TITLE
Feature: `sort-imports` allow to disable sorting for side-effects and styles

### ISSUE THREAD

**Issue by antfu** [azat-io/eslint-plugin-perfectionist#259]
### What rule do you want to change?

sort-imports

### Describe the problem

Side-effect and style imports are often order sensitive, for example:

```ts
import 'some-ui/style.css'
import './main.css'
import './override.css'
```

Currently, the `sort-imports` will always sort them based on their name.

Maybe we could have specify the sorting per-group, for example:

```ts
"perfectionist/sort-imports": ["error", {
  customGroups: {
    value: {
      // Instead of myGroup: ['react', ...] now it can be an object
      myGroup: {
		sort: 'none', // 'alphabetical' | 'line-length' | 'natural' with additiona 'none',
        specifiers: ['react', ...]
      }
    }
  }
]
```

Where I would wish the default groups `side-effects` `side-effect-style` `style` to have `sort: none` by default.

### Code example

-

### Validations

- [X] Read the [docs](https://perfectionist.dev).
- [X] Check that there isn't [already an issue](https://github.com/azat-io/eslint-plugin-perfectionist/issues) that reports the same bug to avoid creating a duplicate.

**Comment by azat-io** [comment]
Thank you for your issue.
I will make this feature soon!

**Comment by azat-io** [comment]
Do you think it might be worth disabling side effect import sorting altogether for security reasons?

**Comment by azat-io** [comment]
Side effect imports sorting is disabled by default.

Fixed in ae02009. Released in [v3.5.0](https://github.com/azat-io/eslint-plugin-perfectionist/releases/tag/v3.5.0).

**Comment by antfu** [comment]
That was fast! Thank you!
